source/interact-data/specify-query.txt

```diff
@@ -0,0 +1,476 @@
+operations on ``Criteria`` objects. These methods take one or more hashes
+of conditions or another ``Criteria`` object as their arguments. The
+``not`` operation has an argument-free version.
+
+The following code demonstrates how to use the logical operations in
+queries:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-logical-ops
+   :end-before: end-logical-ops
+   :language: ruby
+   :dedent:
+
+.. note:: Array Parameters
+
+   To ensure backwards compatibility with earlier {+odm+} versions, the
+   logical operation methods accept arrays of parameters, which are
+   flattened to obtain the criteria.
+   
+   Passing arrays to logical operations is deprecated and may be removed
+   in a future version.
+
+The following queries produce the same conditions:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-logical-and-ops
+   :end-before: end-logical-and-ops
+   :language: ruby
+   :dedent:
+
+Operator Combinations
+~~~~~~~~~~~~~~~~~~~~~
+
+The logical operators have the the same semantics as those from Active
+Record. 
+
+When conditions are specified on the same field multiple times, all
+conditions are added to the criteria, as shown by the queries in the
+following code:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-logical-combination-ops
+   :end-before: end-logical-combination-ops
+   :language: ruby
+   :dedent:
+
+The ``any_of()``, ``none_of()``, ``nor()``, and ``not()`` operations
+behave similarly.
+
+When you use ``and()``, ``or()``, and ``nor()`` logical operators, they
+operate on the criteria built up to that point:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-logical-combination-ops-2
+   :end-before: end-logical-combination-ops-2
+   :language: ruby
+   :dedent:
+
+not() Behavior
+~~~~~~~~~~~~~~
+
+You can use the ``not()`` method without arguments, in which case it
+negates the next condition that is specified. The ``not()`` method can
+be called with one or more hash conditions or ``Criteria`` objects,
+which are all negated and added to the criteria.
+
+The following examples demonstrate the behavior of ``not()``:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-not-logical
+   :end-before: end-not-logical
+   :language: ruby
+   :dedent:
+
+.. note::
+   
+   You cannot use the ``$not`` operator in MongoDB with a string argument.
+   Mongoid uses the ``$ne`` operator to achieve negation:
+
+   .. literalinclude:: /includes/interact-data/query.rb
+      :start-after: start-not-logical-note
+      :end-before: end-not-logical-note
+      :language: ruby
+      :dedent:
+
+Similarly to ``and()``, the ``not()`` operation negates individual
+conditions for simple field criteria. For complex conditions and when a
+field already has a condition defined on it, {+odm+} emulates ``$not``
+by using an ``{'$and' => [{'$nor' => ...}]}`` construct, since MongoDB
```

Comment:
[Style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-since) - 'because' over 'since'
```suggestion
by using an ``{'$and' => [{'$nor' => ...}]}`` construct, because MongoDB
```